packages/flutter/test/widgets/spell_check_test.dart

```diff
@@ -142,15 +142,14 @@ void main() {
     );
 
     expect(textSpanTree, equals(expectedTextSpanTree));
-  });
+  }, variant: const TargetPlatformVariant(<TargetPlatform>{ TargetPlatform.android, TargetPlatform.iOS }));
 
-  test(
-      'buildTextSpanWithSpellCheckSuggestions corrects results when they lag, results text shorter than actual text example',
-      () {
+  testWidgets(
+    'buildTextSpanWithSpellCheckSuggestions corrects results when they lag, results text shorter than actual text example',
+      (WidgetTester tester) async {
     const String text = 'Hello, wrold! Hey';
     const String resultsText = 'Hello, wrold!';
-    const TextEditingValue value = TextEditingValue(
-        text: text, composing: TextRange(start: 14, end: 17));
```

Comment:
Is this case with a composing region tested somewhere else?  If not, maybe we should keep both, one original with the composing region and one your update without composing region?

Same for other tests where the composing region was removed.

Reply:
I am open to this, but I don't think it's necessary. Cases with the composing region are tested in the other tests, and the purpose of the tests where I removed it have to do with correcting the spell check results versus correctly styling the composing region.

Reply:
Ok sounds good to keep it like this if it's covered elsewhere 👍